### PR TITLE
added Quiz Page

### PR DESCRIPTION
**Pull request by Radheshyam1234**
frontQuiz-instruction
frontQuiz-quizPage

### CODE REVIEW

**Review by yashraj2312 — APPROVED** [review]
LGTM

**Comment by KumarVishvas12** [comment]
Code is well structured.
Best practices has been followed. 
You can merge

File: src/App.tsx [hunk 1]
```diff
@@ -1,6 +1,15 @@
...
+  Navbar,
+  LandingPage,
+  SingleCategoryPage,
+  Profile,
+  PrivateRoute,
+  Login,
+  SignUp,
+  QuizContainer,
+} from "./Components";
 import { API_URL } from "./utilities/constants";
```
Comment (parthpandyappp) [inline src/App.tsx:13]: Good use of constants for sensitive information.

File: src/App.tsx [hunk 2]
```diff
@@ -1,6 +1,15 @@
...
 import { useQuizData } from "./Context/QuizContext/QuizDataProvider";
```
Comment (parthpandyappp) [inline src/App.tsx:14]: If you want to avoid writing long lines for imports, you can use a combination of absolute imports and barrels.

**Review by parthpandyappp — COMMENTED** [review]
Good work on Typescript. Have left some comments,  have a look :)

**Pull request merged**